man/crypttab.xml

```diff
@@ -260,6 +260,29 @@
         <xi:include href="version-info.xml" xpointer="v243"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>link-volume-key-to-keyring=</option></term>
+
+        <listitem><para>Specifies the keyring and key description where
```

Comment:
please say *kernel* keyring. since there are other keyrings around, and it really matters that in the docs we clarify this. 

i.e. in my linux system here i have an executable "keyring" installed, and it's not about the kernel keyring at all. frnakly, i have no idea at al what it is, the man page is crap, but we should certainly not making things even more confusing.

maybe add a man page link to keyrings(7)